profiles/nancyKasaudhan20.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/97733705?v=4",
+    "issueId": 1764,
+    "name": "Nancy Kasaudhan",
+    "username": "nancyKasaudhan20"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```